src/test/kotlin/org/rust/ide/intentions/RsIntentionTestBase.kt

```diff
@@ -98,11 +97,9 @@ abstract class RsIntentionTestBase(private val intentionClass: KClass<out Intent
         // Check preview only for intentions from Rust plugin
         if (intentionClass.isSubclassOf(RsElementBaseIntentionAction::class)) {
             if (previewExpected) {
-                checkPreviewAndLaunchAction(intention, preview)
+                checkPreviewAndLaunchAction(myFixture, intention, preview)
```

Comment:
Maybe convert it into an extension method of `CodeInsightTestFixture`? not to pollute namespace by top-level functions